mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased changes
+## v1.0.1 (August 31, 2023)
 
 * Added new metrics for object writes, IO sizes, file handles, and directory operations. The existing `fuse.bytes_read` metric has been renamed to `fuse.total_bytes` and is now keyed by operation (`read`/`write`).
```

Comment:
We should add these PRs to the changelog:
- https://github.com/awslabs/mountpoint-s3/pull/475
- https://github.com/awslabs/mountpoint-s3/pull/467
- https://github.com/awslabs/mountpoint-s3/pull/489